openapi/build.sbt

```diff
@@ -1,61 +1,52 @@
 import EndpointsSettings._
 import LocalCrossProject._
 
-val testsuite = LocalProject("testsuite")
+lazy val `json-schemaJS` = LocalProject("json-schemaJS")
+lazy val `json-schemaJVM` = LocalProject("json-schemaJVM")
 
-val `json-schema` =
-  crossProject.crossType(CrossType.Pure).in(file("json-schema"))
-    .settings(publishSettings ++ `scala 2.10 to 2.12`: _*)
-    .settings(
-      name := "endpoints-openapi-json-schema",
-      addScalaTestCrossDependency
-    )
-
-val `json-schema-js` = `json-schema`.js
-val `json-schema-jvm` = `json-schema`.jvm
-
-val `json-schema-generic` =
+lazy val `json-schema-generic` =
   crossProject.crossType(CrossType.Pure).in(file("json-schema-generic"))
     .settings(publishSettings ++ `scala 2.11 to 2.12`: _*)
     .settings(
       name := "endpoints-openapi-json-schema-generic",
       libraryDependencies += "com.chuusai" %%% "shapeless" % "2.3.2",
       addScalaTestCrossDependency
     )
-    .dependsOn(`json-schema`)
+    .dependsOnLocalCrossProjects("json-schema")
+    .dependsOn(`openapi` % Test)
 
-val `json-schema-generic-js` = `json-schema-generic`.js
-val `json-schema-generic-jvm` = `json-schema-generic`.jvm
+lazy val `json-schema-generic-js` = `json-schema-generic`.js
+lazy val `json-schema-generic-jvm` = `json-schema-generic`.jvm
 
-val `json-schema-circe` =
+lazy val `json-schema-circe` =
   crossProject.crossType(CrossType.Pure).in(file("json-schema-circe"))
     .settings(publishSettings ++ `scala 2.10 to 2.12`: _*)
     .settings(
       name := "endpoints-openapi-json-schema-circe",
```

Comment:
This one is not coupled with OpenAPI, is it? I think we should find another name for it.

Reply:
renamed and moved